### ISSUE TITLE
Clickable URLs

### ISSUE THREAD

**Issue by svandragt** [johnbillion/query-monitor#601]
Is it possible to have the URLs in the Query Monitor main panel clickable? For example when a `/wp-admin` call has an **HTTP API Calls** fatal error the URL is shown in the main panel but the source of the problem is not revealed until that URL is opened in a new tab. 

This trips up junior developers who get stuck trying to diagnose issues, so in my opinion it'll be a nice refinement for everyone :)

**Comment by johnbillion** [comment]
Yeah the main reason I haven't done this so far is it doesn't make sense to provide the same for non-GET requests, but I guess that's no reason not to make the URL clickable for GET requests. I think the best thing is to add a button at the end of the URL.
